site2/docs/deploy-bare-metal-multi-cluster.md

```diff
@@ -62,45 +62,45 @@ The Pulsar binary package initially contains the following directories:
 
 Directory | Contains
 :---------|:--------
-`bin` | Pulsar's [command-line tools](reference-cli-tools.md), such as [`pulsar`](reference-cli-tools.md#pulsar) and [`pulsar-admin`](reference-pulsar-admin.md)
+`bin` | [Command-line tools](reference-cli-tools.md) of Pulsar, such as [`pulsar`](reference-cli-tools.md#pulsar) and [`pulsar-admin`](reference-pulsar-admin.md)
 `conf` | Configuration files for Pulsar, including for [broker configuration](reference-configuration.md#broker), [ZooKeeper configuration](reference-configuration.md#zookeeper), and more
 `examples` | A Java JAR file containing example [Pulsar Functions](functions-overview.md)
-`lib` | The [JAR](https://en.wikipedia.org/wiki/JAR_(file_format)) files used by Pulsar
+`lib` | The [JAR](https://en.wikipedia.org/wiki/JAR_(file_format)) files that Pulsar uses 
 `licenses` | License files, in `.txt` form, for various components of the Pulsar codebase
 
-These directories will be created once you begin running Pulsar:
+The following directories are created once you begin running Pulsar:
 
 Directory | Contains
 :---------|:--------
-`data` | The data storage directory used by ZooKeeper and BookKeeper
+`data` | The data storage directory that ZooKeeper and BookKeeper use
 `instances` | Artifacts created for [Pulsar Functions](functions-overview.md)
-`logs` | Logs created by the installation
+`logs` | Logs that the installation creates
 
 
-## Deploying ZooKeeper
+## Deploy ZooKeeper
 
 Each Pulsar instance relies on two separate ZooKeeper quorums.
 
-* [Local ZooKeeper](#deploying-local-zookeeper) operates at the cluster level and provides cluster-specific configuration management and coordination. Each Pulsar cluster needs to have a dedicated ZooKeeper cluster.
-* [Configuration Store](#deploying-configuration-store) operates at the instance level and provides configuration management for the entire system (and thus across clusters). The configuration store quorum can be provided by an independent cluster of machines or by the same machines used by local ZooKeeper.
+* [Local ZooKeeper](#deploying-local-zookeeper). It operates at the cluster level and provides cluster-specific configuration management and coordination. Each Pulsar cluster needs to have a dedicated ZooKeeper cluster.
```

Comment:
Check the links when you update the titles.
For example, since you have update the title from "Deploying local ZooKeeper" to "Deploy local ZooKeeper", the link should be update accordingly as "[Local ZooKeeper](#deploy-local-zookeeper)".
Check all similar cases.